src/main/scala/it/pagopa/interop/tenantmanagement/api/impl/TenantApiServiceImpl.scala

```diff
@@ -113,6 +113,38 @@ class TenantApiServiceImpl(
     }
   }
 
+  override def addTenantMail(tenantId: String, mailSeed: MailSeed)(implicit
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    contexts: Seq[(String, String)]
+  ): Route = {
+    val operationLabel = s"Adding Mail ${mailSeed.id} with address ${mailSeed.address} to Tenant $tenantId"
```

Comment:
The email could be considered a sensitive information, it's better to not include it in the logs
```suggestion
    val operationLabel = s"Adding Mail ${mailSeed.id} to Tenant $tenantId"
```